### PR TITLE
Add Unstoppable wallet: an open-source blockchain wallet.

### PR DESCRIPTION
**Pull request by esengulov**
Added a link to Decentralized wallet for Bitcoin, Ethereum, EOS, Binance Chain, Bitcoin Cash and DASH blockchains

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -17870,6 +17870,28 @@
...
+      "description": "A fully decentralized Bitcoin, Ethereum, EOS, Binance Chain, Bitcoin Cash and DASH currency wallet",
+      "category-ids": [
+        "cryptocurrency"
+      ],
+      "source": "https://github.com/horizontalsystems/unstoppable-wallet-ios",
+      "itunes": "https://itunes.apple.com/app/bank-bitcoin-wallet/id1447619907?ls=1&mt=8",
+      "license": "mit",
+      "stars": 89,
+      "tags": [
+        "swift",
```
Comment (dkhamsing) [inline contents.json]: Can you remove the `,` at the end

File: contents.json [hunk 2]
```diff
@@ -17870,6 +17870,28 @@
...
+      ],
+      "lang": "eng",
```
Comment (dkhamsing) [inline contents.json]: I don't think this is necessary, can you remove this line with `lang` please

File: contents.json (hunk 1 above)
Comment (esengulov) [reply contents.json]: done!

File: contents.json (hunk 2 above)
Comment (esengulov) [reply contents.json]: done!

**Comment by esengulov** [comment]
all requested changes made!
thank you ;)

**Comment by dkhamsing** [comment]
Thank you

**Pull request merged**